flask_mailgun.py

```diff
@@ -49,21 +58,36 @@
 """
 
 
-class Mailgun:
+default_api_error_template = (
+    'Mailgun API {method} at {url} failed '
+    'with status code {status_code}:\n{text}')
 
+
+class Mailgun(object):
     """Mailgun integration for Flask."""
 
     def __init__(self, app=None):
         self.debug = None
         self.domain = None
         self.key = None
+
+        self.api_url_template = None
+        self.debug_template = None
+        self.logging_template = None
+
+        self.error_subject_withexcinfo_template = None
+        self.error_subject_withoutexcinfo_template = None
+        self.error_template = None
+
+        self.api_error_template = None
+
         if app is not None:
             self.init_app(app)
 
     def init_app(self, app):
         """Inititialize the extension.
 
-        There are two configuration options for authentization:
+        There are two configuration options for authentication:
```

Comment:
Good catch, I mixed up authentication with authorization in my head into one word. Perhaps this should be in a separate commit?